build.gradle

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "uk.nhs.hee.tis.trainee"
-version = "1.8.3"
+version = "1.8.4"
```

Comment:
I'd argue this should be a MINOR version bump

Reply:
Fair enough 👍